src/components/UserPortal/OrganizationCard/OrganizationCard.tsx

```
@@ -100,96 +131,113 @@ function organizationCard(props: InterfaceOrganizationCardProps): JSX.Element {
-                {tCommon('admins')}: <span>{props.admins?.length}</span>
-              </div>
-              <div>
-                {tCommon('members')}: <span>{props.members?.length}</span>
-              </div>
-            </h6>
-          </div>
+    <div className={styles.orgCard}>
+      <div className={styles.innerContainer}>
+        <div className={styles.orgImgContainer}>
+          {image ? (
+            <img src={image} alt={`${name} image`} />
```

Biome (1.9.4)

[error] Avoid the words "image", "picture", or "photo" in img element alt text.

Screen readers announce img elements as "images", so it is not necessary to redeclare this in alternative text.

(lint/a11y/noRedundantAlt)